frontend/src/console/components/CoursesManage/styles.scss

```
@@ -0,0 +1,80 @@
+@import "~styles/theme";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+.courses-page {
+    // Only put this because the page didn't pick up the right font family.
+    font-family: 'Montserrat';
+
+    h2 {
+        font-family: 'Montserrat' !important;
+        font-size: 30px;
+        font-weight: 600;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        color: #000000;
+    }
+
+    p {
+        margin-top: 20px;
+    }
+
+    .row {
+        margin-bottom: 5px;
+    }
+
+    .numberCircle {
```

Rename to match class name pattern
```suggestion
    .number-circle {
```